sopel/bot.py

```diff
@@ -417,6 +422,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
                 reply_to = self._trigger.nick
             self._bot.reply(message, destination, reply_to, notice)
 
+        def kick(self, message, channel, nick):
```

Comment:
`message` should be the last argument, and default to `''` or `None`, so it can be omitted. The `KICK` IRC command itself only requires `channel` and `nick`.